src/modules/skinManager/src/skinManager.cpp

```diff
@@ -593,10 +593,10 @@ bool skinManager::identifyCommand(Bottle commandBot, SkinManagerCommand &com, Bo
 
 bool skinManager::updateModule() { 
     double avgTime, stdDev, period;
-    period = myThread->getRate();
-    myThread->getEstPeriod(avgTime, stdDev);
+    period = myThread->getPeriod();
```

Comment:
Here I preferred going with `[s]` instead of transforming in `[ms]` not knowing what may happen to the standard deviation.

However, computations should remain self-consistent.

Reply:
Agreed, this are just used for some debug prints.